frontend/src/Components/Page/Header/SeriesSearchInput.js

```diff
@@ -161,13 +161,12 @@ class SeriesSearchInput extends Component {
       return;
     }
 
-    // If an suggestion is not selected go to the first series,
-    // otherwise go to the selected series.
-
-    if (highlightedSuggestionIndex == null) {
-      this.goToSeries(suggestions[0]);
-    } else {
+    // If a suggestion is highlighted, go to that series
+    // otherwise search for a new series
+    if (highlightedSuggestionIndex) {
```

Comment:
`null`, `undefined` and `0` will all fall through this check, which means the first item will never be selectable.

Something like this would be more appropriate.

```
    if (highlightedSuggestionIndex == null) {
      this.props.onGoToAddNewSeries(value);
    } else {
      this.goToSeries(suggestions[highlightedSuggestionIndex]);
    }
```

Reply:
Fixed 😄